extensions/azure/blob/reader/src/main/java/org/eclipse/dataspaceconnector/reader/azure/blob/BlobStoreReader.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.dataspaceconnector.reader.azure.blob;
+
+import org.eclipse.dataspaceconnector.common.azure.BlobStoreApi;
+import org.eclipse.dataspaceconnector.spi.result.Result;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataAddress;
+import org.eclipse.dataspaceconnector.transfer.inline.spi.DataReader;
+
+import java.io.ByteArrayInputStream;
+
+public class BlobStoreReader implements DataReader {
+
+    private final BlobStoreApi blobStoreApi;
+
+    public BlobStoreReader(BlobStoreApi blobStoreApi) {
+        this.blobStoreApi = blobStoreApi;
+    }
+
+    @Override
+    public boolean canHandle(String type) {
+        return "AzureStorage".equals(type);
+    }
+
+    @Override
+    public Result<ByteArrayInputStream> read(DataAddress source) {
+        var account = source.getProperty("account");
```

Comment:
This was not introduced by you, but could we use `AzureBlobStoreSchema` to access those constants?